### DISCUSSION TITLE
[question] How to get arn address

### DISCUSSION THREAD

**Discussion by invent-lwj** [minio/minio#12437] [Q&A] [answered]
I establish minio cluster on local machine, i want to get event of a bucket, this command must input ARN, how to get it? Thanks!
```
EXAMPLES:
  1. Enable bucket notification with a specific ARN
    $ mc event add myminio/mybucket arn:aws:sqs:us-west-2:444455556666:your-queue
```

**Comment by harshavardhana** [comment, marked as answer]
@invent-lwj  follow the docs here https://docs.min.io/docs/minio-bucket-notification-guide.html
